extensions/escape-game/standalone.mjs

```diff
@@ -390,6 +390,13 @@ class Standalone {
     }
   }
 
+  /**
+   * The secret function to give a "certificate" item to the player
+   */
+   async c2s_getCertificateItem(player) {
+    return await this.helper.callS2sAPI('items', 'AddItem', player.playerID, 'certificate', 1, 1);
```

Comment:
There are no restrictions on the client side calling this, we should add a flag or secret to make sure only those who've completed the mission get the item. It is recommended that the flag be configurable from config file.